README.md

```diff
@@ -1,5 +1,20 @@
 # Eightshift frontend libs
 
-A collection of useful frontend utility modules.
+This lib is used to work in correlation with [Eightshift Libs](https://github.com/infinum/eightshift-libs/) and [Eightshift Boilerplate](https://github.com/infinum/eightshift-boilerplate-internal) by hosting all frontend parts for your project.
```

Comment:
```suggestion
This lib is intended to work in correlation with [Eightshift Libs](https://github.com/infinum/eightshift-libs/) and [Eightshift Boilerplate](https://github.com/infinum/eightshift-boilerplate-internal) by hosting all frontend parts for your project.
```
I'm a bit confused, I know we discussed that we'll use only one of the boilerplates in the end but I'm not 100% sure which one. The wp-boilerplate has more starts, so we should use that one, no?